libbpfgo/libbpfgo.go

```diff
@@ -366,6 +366,25 @@ func (b *BPFMap) SetPinPath(pinPath string) error {
 	return nil
 }
 
+func (b *BPFMap) Resize(maxEntries uint32) error {
+	cs := C.CString(b.name)
+	bpfMap := C.bpf_object__find_map_by_name(b.module.obj, cs)
+	errC := C.bpf_map__resize(bpfMap, C.uint(maxEntries))
+	C.free(unsafe.Pointer(cs))
+	if errC != 0 {
+		return fmt.Errorf("failed to resize map %s to %v", b.name, maxEntries)
+	}
+	return nil
+}
+
+func (b *BPFMap) GetMaxEntries() uint32 {
```

Comment:
I'd like to start adding GoDoc strings to all the libbpfgo API, mind starting us off with these two?